### PR TITLE
en: update docs about data mount path for DM-worker

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added, or deleted? (Required)

update docs about data mount path for DM-worker

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1218
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by csuzhangxc — APPROVED** [review]
LGTM

**Comment by qiancai** [comment]
@Joyinqin PTAL. Thanks!

File: en/deploy-tidb-dm.md [hunk 1]
```diff
@@ -217,22 +217,30 @@ kubectl exec -ti ${dm_cluster_name}-dm-master-0 -n ${namespace} - /bin/sh
...
 ### Configure migration tasks
 
 1. Edit task configuration file `task.yaml`, which can refer to [Configure the data migration task](https://docs.pingcap.com/tidb-data-migration/v2.0/migrate-data-using-dm#step-4-configure-the-data-migration-task).
 
 2. Configure the `target-database.host` in `task.yaml` as the TiDB host address that the Kubernetes cluster can access internally. If the cluster is deployed by TiDB Operator, configure the host as `${tidb_cluster_name}-tidb.${namespace}`.
 
+3. In the `task.yaml` file, take the following steps:
+
+    - Add the `loaders.${customized_name}.dir` field as the import and export directory for the full volume data, where `${customized_name}` is a name that you can customize. 
+    - Configure `loaders.${customized_name}.dir` as the subdirectory of the persistent volume in the Pod `/var/lib/dm-worker` directory. For example, `/var/lib/dm-worker/dumped_data`.
```
Comment (Joyinqin) [inline en/deploy-tidb-dm.md:241]:
```suggestion
    - Configure the `loaders.${customized_name}.dir` field as the subdirectory of the persistent volume in the Pod `/var/lib/dm-worker` directory. For example, `/var/lib/dm-worker/dumped_data`.
```

File: en/deploy-tidb-dm.md [hunk 2]
```diff
@@ -217,22 +217,30 @@ kubectl exec -ti ${dm_cluster_name}-dm-master-0 -n ${namespace} - /bin/sh
...
+    - Reference `${customized_name}` in the instance configuration. For example, `mysql-instances[0].loader-config-name: "{customized_name}"`.
```
Comment (Joyinqin) [inline en/deploy-tidb-dm.md:242]:
```suggestion
    - Reference the `${customized_name}` field in the instance configuration. For example, `mysql-instances[0].loader-config-name: "{customized_name}"`.
```

**Review by Joyinqin — APPROVED** [review]
minor comments, rest /lgtm

File: en/deploy-tidb-dm.md (hunk 2 above)
Comment (qiancai) [reply en/deploy-tidb-dm.md:242]: As we need to reference only the value of the field, not the field, so I will skip this comment.

**Pull request merged**